compose/opensearch.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  opensearch-node:
+    image: docker.io/opensearchproject/opensearch:latest
+    hostname: opensearch-node
+    environment:
+      discovery.type: single-node
+      # Reference: https://opensearch.org/docs/latest/security/configuration/demo-configuration/#setting-up-a-custom-admin-password
+      OPENSEARCH_INITIAL_ADMIN_PASSWORD: password4Opense@rch
+      OPENSEARCH_JAVA_OPTS: >
+        -Dcom.sun.management.jmxremote.autodiscovery=true
+        -Dcom.sun.management.jmxremote
+        -Dcom.sun.management.jmxremote.port=9292
+        -Dcom.sun.management.jmxremote.rmi.port=9292
+        -Djava.rmi.server.hostname=opensearch-node
+        -Dcom.sun.management.jmxremote.authenticate=false
+        -Dcom.sun.management.jmxremote.ssl=false
+        -Dcom.sun.management.jmxremote.local.only=false
+    volumes:
+      - opensearch-data:/usr/share/opensearch/data
+    ports:
+      - 9200:9200 # REST API
+      - 9600:9600 # Performance Analyzer
```

Comment:
Is this something that should also be in `expose`? I wasn't able to get an HTTP(S) response on this port from my host machine anyway.

Reply:
Hmm, I cant seem to have anything returned either...Let me check...

Reply:
Hmm, maybe we need to enable some settings.

Reply:
I think we can just remove it, not sure how much use we really have for this port unless it reports some metrics that are different but comparable to JMX/JFR data?

Reply:
I guess it might be valuable to do some comparison to JFR? Looks like some metrics are similar https://opensearch.org/docs/latest/monitoring-your-cluster/pa/index/#example-api-query-and-response

Reply:
I am still exploring and lacking some contexts. Let's remove it for now I guess. I will do some more research as I learn and add it back later :D